pkg/doc/did/doc.go

```diff
@@ -400,13 +399,15 @@ func populateVerificationMethods(context string, rawVerificationMethods []interf
 }
 
 // getVerificationMethods gets verification methods from raw data
-// TODO support relative DID URLs (https://www.w3.org/TR/did-core/#relative-did-urls)
-//  (https://github.com/hyperledger/aries-framework-go/issues/1662)
-func getVerificationMethods(context string, rawVerificationMethod interface{},
-	pks []PublicKey, relationship VerificationRelationship) ([]VerificationMethod, error) {
+func getVerificationMethods(doc *Doc, rawVerificationMethod interface{},
+	relationship VerificationRelationship) ([]VerificationMethod, error) {
+	// context, docID string
+	pks := doc.PublicKey
+	context := doc.Context[0]
```

Comment:
is Context[0] always going to exist? if not, consider adding check if Context is empty

Reply:
If the `Doc` is unmarshaled from bytes (using `ParseDocument()`) then it must contain at least one context, so it's safe.
But in the case of manual construction of `Doc` structure it's possible... It relies on user's discipline. :)